recipes/libdatachannel/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import copy, get, rm, rmdir, export_conandata_patches, apply_conandata_patches
+from conan.tools.microsoft import is_msvc
+from conan.tools.apple import fix_apple_shared_install_name
+
+required_conan_version = ">=2.1"
+
+class libdatachannelConan(ConanFile):
+    name = "libdatachannel"
+    description = "C/C++ WebRTC network library featuring Data Channels, Media Transport, and WebSockets."
+    license = "MPL-2.0"
+    topics = ("webrtc")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/paullouisageneau/libdatachannel"
+    settings = "os", "compiler", "build_type", "arch"
+    package_type = "library"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_websocket": [True, False],
+        "with_nice": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_websocket": True,
+        "with_nice": False
+    }
+
+    implements = ["auto_shared_fpic"]
+
+    def requirements(self):
+        self.requires("openssl/[>=1.1]")
+        self.requires("plog/1.1.10")
+        self.requires("usrsctp/0.9.5.0")
+        self.requires("libsrtp/2.6.0")
+        self.requires("nlohmann_json/3.11.3")        
+        if self.options.with_nice:
+            self.requires("libnice/0.1.21")
+        else:
+            self.requires("libjuice/1.5.7")
+
+    def validate(self):
+        check_min_cppstd(self, 11)
```

Comment:
it looks like this is actually 17? unless it affects one of the dependencies

Reply:
yes it is required really